tests/lib/prepare.sh

```diff
@@ -628,6 +746,15 @@ EOF
         IMAGE_CHANNEL="$GADGET_CHANNEL"
     fi
 
+    if is_core20_system; then
+        snap download --channel="20/$KERNEL_CHANNEL" pc-kernel
```

Comment:
can we rename this with --basename so that we don't have multiple files here? i.e. right now this downloads as `pc-kernel_239.snap`, then when we repack we repack into a different dir a snap named `pc-kernel_5.4.12.15.snap`, so if the directories get changed somehow such that now both snaps are put into the same dir, we would end up with `pc-kernel_*.snap` resolved to either multiple snaps or possibly the wrong one depending on the context that `pc-kernel_*.snap` is used in.

Reply:
I made this change here